docs/cis_self_assessment.md

```diff
@@ -1555,7 +1555,7 @@ Ensure that the `client-cert-auth` field is set to `true` (Scored)
 etcd is a highly-available key value store used by Kubernetes deployments for persistent storage of all of its REST API objects. These objects are sensitive in nature and should be accessible only by authenticated etcd peers in the etcd cluster.
 </details>
 
-**Result:** **Not Applicable**
+**Result:** Not Applicable
```

Comment:
glad you removed the ** here, because this should be "pass" not "Not applicable"

Reply:
2.4 has an erroneous * too